torch/testing/_creation.py

```diff
@@ -7,6 +7,11 @@
 import math
 import collections.abc
 
+# Used by make_tensor for generating complex tensor.
+complex_to_corresponding_float_type_map = {torch.complex32: torch.float16,
```

Comment:
Was planning to put this in `torch.testing._internal.common_utils.py` but it uses `make_tensor`. Wasn't ready to fight circular dependency. So have kept it here.